Docs/README.md

```diff
@@ -15,11 +18,12 @@ This is an an attempt at descripting the different programs and analyses that co
    - gene body coverage
    - read distribution
    - junction annotation
-* dupRadar
-* preseq - subread featureCounts - gene counts, biotype counts, rRNA estimation.
+* dupRadar - investigate duplication problems
+* preseq - 
+* subread featureCounts - gene counts, biotype counts, rRNA estimation.
 * String Tie - FPKMs for genes and transcripts
 * edgeR - create MDS plot and sample pairwise distance heatmap / dendrogram
-* MultiQC
+* MultiQC - aggregate report
 
 ## FastQC
```

Comment:
Documentation link for FastQC is unofficial - better to link to the official (and up to date) help pages: http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/